src/app/api/auth/[...nextauth]/options.ts

```diff
@@ -1,76 +1,123 @@
 import { NextAuthOptions } from "next-auth";
 import CredentialsProvider from "next-auth/providers/credentials";
-import bcrypt from "bcryptjs"
+import GoogleProvider from "next-auth/providers/google";
+import bcrypt from "bcryptjs";
 import dbConnect from "@/lib/dbConnect";
 import UserModel from "@/model/User";
 
 export const authOptions: NextAuthOptions = {
-    providers: [
-        CredentialsProvider({
-            id: "credentials",
-            name: 'Credentials',
-            credentials: {
-                email: { label: "Email", type: "text" },
-                password: { label: "Password", type: "password" }
-            },
-            async authorize(credentials: any): Promise<any> {
-                await dbConnect()
-                try {
-                    const user = await UserModel.findOne({
-                        $or: [
-                            { email: credentials.identifier },
-                            { username: credentials.identifier }
-                        ]
-                    })
-
-                    if (!user) {
-                        throw new Error("No user found with this email")
-                    }
-
-                    if (!user.isVerified) {
-                        throw new Error("Please verify your account before login")
-                    }
-
-                    const isPasswordCorrect = await bcrypt.compare(credentials.password, user.password)
-                    if (isPasswordCorrect) {
-                        return user
-                    } else {
-                        throw new Error("Incorrect password")
-                    }
-                } catch (err: any) {
-                    throw new Error(err);
-
-                }
+  providers: [
+    // Google Provider
+    GoogleProvider({
+        clientId: process.env.GOOGLE_CLIENT_ID ?? "",
+        clientSecret: process.env.GOOGLE_CLIENT_SECRET ?? "",
```

Comment:
Remove ?? "" 

because this is giving error as

[next-auth][error][SIGNIN_OAUTH_ERROR] 
https://next-auth.js.org/errors#signin_oauth_error client_id is required {
  error: {
    message: 'client_id is required',
    stack: 'TypeError: client_id is required\n' +
      '    at new Y (/var/task/.next/server/chunks/830.js:30:53772)\n' +
      '    at new <anonymous> (/var/task/.next/server/chunks/830.js:39:22174)\n' +
      '    at o (/var/task/.next/server/chunks/830.js:6:126196)\n' +
      '    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)\n' +
      '    at async s (/var/task/.next/server/chunks/830.js:6:118226)\n' +
      '    at async Object.s (/var/task/.next/server/chunks/830.js:30:8621)\n' +
      '    at async _ (/var/task/.next/server/chunks/830.js:6:103586)\n' +
      '    at async a (/var/task/.next/server/chunks/830.js:30:18306)\n' +
      '    at async e.length.t (/var/task/.next/server/chunks/830.js:30:19717)\n' +
      '    at async /var/task/node_modules/next/dist/compiled/next-server/app-route.runtime.prod.js:6:36258',
    name: 'TypeError'
  },
  providerId: 'google',
  message: 'client_id is required'
}